lib/routing.py

```diff
@@ -104,15 +114,22 @@ def add_route(self, func, pattern):
 
     def run(self, argv=sys.argv):
         if len(argv) > 2:
-            self.args = parse_qs(argv[2].lstrip(b'?'))
+            try:
+                self.args = parse_qs(argv[2].lstrip(b'?'))
+            except TypeError:
+                self.args = parse_qs(argv[2].lstrip('?'))
```

Comment:
Not sure why the `b` is there, it doesn't seem to be necessary. So this `try` can be removed as well